### PR TITLE
chore(css): Adjust bottom margin of videos followed by paragraphs in the documentation

### PR DESCRIPTION
**Pull request by agneszitte**
## PR Type

What kind of change does this PR introduce?

- Documentation layout changes (CSS)

## Description

Adjust the bottom margin of videos for this specific case where they are followed by paragraphs in the documentation

## Before / After

| Before  | After |
| ------------- | ------------- |
| Before | After |

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Comment by agneszitte** [comment]
@Mergifyio backport release/stable/5.2

**Comment by agneszitte** [comment]
@Mergifyio backport release/stable/5.3

File: doc/templates/uno/main.scss [hunk 1]
```diff
@@ -489,6 +489,11 @@ h5 {
   }
 }
 
+// Specific case for paragraphs under embedded videos
+.embeddedvideo + p {
```
Comment (ajpinedam) [inline doc/templates/uno/main.scss:493]: 🚀

**Review by ajpinedam — APPROVED** [review]

**Review by morning4coffe-dev — APPROVED** [review]
Great improvement! 🚀

**Pull request merged**